src/ImgixTag.js

```diff
@@ -187,14 +187,16 @@ var ImgixTag = (function () {
 
   ImgixTag.prototype.sizes = function () {
     var existingSizes = this.el.getAttribute('sizes');
+    var dataSizes = this.el.getAttribute('ix-sizes');
     const el = this.el;
     const _window = this.window;
 
-    if (existingSizes && existingSizes !== 'auto') {
-      return existingSizes;
-    } else if (existingSizes === 'auto') {
-      return autoSize.updateOnResize({ el, existingSizes, _window });
+    if (existingSizes || dataSizes !== 'auto') {
+      return existingSizes ? existingSizes : '100vw';
+    } else if (dataSizes === 'auto') {
+      return autoSize.updateOnResize({ el, existingSizes, dataSizes, _window });
```

Comment:
The `&&` here was changed to `||` because if `dataSizes` is ever `false` we should not be automatically sizing the element.

Here we also add the ternary operator in order to ensure that, whenever `existingSizes` evaluates to `false`, we return the browser default `100vw`. This can happen when a user stubs an image tag or incorrectly sets the attribute.